database/seeders/Test/TestSeeder.php

```diff
@@ -27,18 +27,18 @@ public function run(): void
         ]);
 
         $teamP1 = Team::create([
-            'id' => 3,
+            'id' => 1,
             'name' => 'P1 Test Team',
         ]);
         $teamP2 = Team::create([
-            'id' => 4,
+            'id' => 2,
             'name' => 'P1 Test Team 2',
         ]);
 
         $program->teams()->sync([$teamP1->id, $teamP2->id]);
 
         $nonProgramTeam = Team::create([
-            'id' => 5,
+            'id' => 3,
```

Comment:
As this seeder is shared with the TestRealForms setup, I would check this also works with that. 

Given the last testing would have been done with the real forms, it's likely this works with that - it might have been 'fixed' to work with the TestRealForms, which then broke it with the TestMiniForms.